tests/noto_fonts_for_android_test.py

```diff
@@ -79,6 +81,17 @@ def _weight(font: ttLib.TTFont) -> Tuple[int, int, int]:
   return (os2_weight, os2_weight, os2_weight)
 
 
+def _psname(font_el) -> str:
+  # Not every font element will have postScriptName tag. If it's not present then it is assumeed
+  # that it's the filename less extension
+  psn = font_el.attrib.get("postScriptName","")
+  if len(psn) == 0:
+    path = str(_font_path(font_el))
+    assert len(path) > 0
+    psn = path[path.rfind('/') + 1 : path.rfind('.')]
```

Comment:
better to leave _font_path a path and use it's apis. In this case the equivalent is something like:

```
path = _font_path(font_el)
assert path.is_file(), f"{path} missing"
return path.stem
```